assets/Product.wxs

```diff
@@ -157,16 +162,19 @@
             <Component Id="ProductVersionFolder" Guid="{e1a7f05e-0cd6-4227-80a8-e4fb311f045c}">
               <CreateFolder/>
             </Component>
+            <Component Id="Telemetry" Guid="{80520f20-471d-4d08-adc8-bab637627b39}" KeyPath="yes">
+              <Environment Id="POWERSHELL_DISTRIBUTION_CHANNEL" Action="create" Name="POWERSHELL_DISTRIBUTION_CHANNEL" Permanent="no" System="yes" Value="MSI:[WINDOWS_PRODUCT_NAME]"/>
```

Comment:
The other variables are delimited with a `-`

Reply:
You mean the Id?  I can switch it to PascalCasing to be consistent.  The Name has to be underscores.